README.developers.md

```diff
@@ -385,6 +385,191 @@ Another reason jobs may not start is if there is a large backlog of jobs
 running, there may be no runners left to start.  In this case, someone with
 Kokoro management rights may need to terminate stale jobs, or wait for job
 timeouts.
+### Parallesim Startegy for vtr_reg_nightly: 
```

Comment:
1. I think this whole description should be moved to the end of the README in https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/tasks/regression_tests/README.md since this is a pretty detailed discussion of optimizing regtest run time. This doc is an overall vtr developer tutorial, so it mostly focuses on how to run tests, rather than the infrastructure of how to optimize run time.

2. Both that document and this document refer to vtr_reg_nightly as a single command you can run.  That will have to be updated.

3. Similarly, the cpu time limits for vtr_reg_nightly mentioned earlier in this doc will have to be updated to instead mention sub-suites that together form vtr_reg_nightly and what the cpu target for them is.